docs/contribute.rst

```diff
@@ -0,0 +1,82 @@
+Contributing Guidelines
+=======================
+
+We welcome contributions! We use `Github projects`_ to organize our ticket workflow. If there's a particular issue you would like to work on and it isn't already assigned, feel free to assign it to yourself and start work! If you just want to lend a hand, check out the current project and choose one of the tickets from the backlog.
+
+Once you're done writing code, you will need to open a pull request with your changes. In order to be merged, pull requests must fulfill the following requirements:
+
+- All new code must have tests.
+- All tests must be passing.
+- Any relevant documentation has been updated.
+
+If you need help with something, that's totally fine. Do what you can and then ask for what you need, either through your PR or on gitter_. Just be aware that there may be a delay before someone comes along who has time to provide it.
+
+Once your pull request is complete, one of the core contributors will review it and give feedback or merge as appropriate.
+
+If you have any questions or just want to discuss the project, chat with us on gitter_.
+
+.. _gitter: https://gitter.im/django-graph-api/Lobby
+.. _Github projects: https://github.com/melinath/django-graph-api/projects
+
+Code of conduct
+---------------
+
+This project adheres to and supports the `Django Code of Conduct`_.
+
+.. _Django Code of Conduct: https://www.djangoproject.com/conduct/
+
+Style guide
+-----------
+
+This project uses the `Django coding style guide`_.
+
+.. _Django coding style guide: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/
+
+
+Start developing
+================
```

Comment:
For some reason this is showing up at the same level as `Contributing Guidelines` in the sidebar, but doesn't get highlighted if I select it.